hp-ux/generate_wazuh_packages.sh

```diff
@@ -267,6 +309,7 @@ main() {
 
     if [[ "${build_env}" = "yes" ]]; then
         build_environment || exit 1
+        exit 0
```

Comment:
This makes it impossible to continue the execution of the script if it is provisioned, right?

Reply:
Done in [7a4ac23](https://github.com/wazuh/wazuh-packages/pull/2173/commits/7a4ac2321412526f3647b0c904c676267def5d8b)